src/core/observables/PidObservable.hpp

```diff
@@ -33,6 +33,8 @@
 
 namespace Observables {
 
+using PartRefSpan = Utils::Span<std::reference_wrapper<const Particle>>;
```

Comment:
Wouldn't a name like `ParticleReferenceRange` or `ConstParticleReferenceRange` that tells something about the concept be more useful, it would also be more in line with`ParticleRange` from core.